src/videoframeprocessor/DefaultVideoFrameProcessorPipeline.ts

```diff
@@ -144,6 +144,7 @@ export default class DefaultVideoFrameProcessorPipeline implements VideoFramePro
     this.inputVideoStream = inputMediaStream;
     const settings = this.inputVideoStream.getVideoTracks()[0].getSettings();
     this.logger.info(`processing pipeline input stream settings ${JSON.stringify(settings)}`);
+    this.framerate = settings.frameRate;
```

Comment:
This does have a 'surprise' performance impact to any customers with input > 15 FPS currently using the pipeline. But i think, at least with V2 bblur pipeline (assuming that is the main usage of this code) that the increase is reasonable, and this is a more 'expected' behavior.